src/patternfly/_patternfly-charts.scss

```diff
@@ -0,0 +1,53 @@
+/* stylelint-disable */
+:root {
+  // Chart colors
+
+  // blue
+  --pf-chart-color-blue-100: #{$pf-chart-color-blue-100};
+  --pf-chart-color-blue-200: #{$pf-chart-color-blue-200};
+  --pf-chart-color-blue-300: #{$pf-chart-color-blue-300};
+  --pf-chart-color-blue-400: #{$pf-chart-color-blue-400};
+  --pf-chart-color-blue-500: #{$pf-chart-color-blue-500};
+
+  // green
+  --pf-chart-color-green-100: #{$pf-chart-color-green-100};
+  --pf-chart-color-green-200: #{$pf-chart-color-green-200};
+  --pf-chart-color-green-300: #{$pf-chart-color-green-300};
+  --pf-chart-color-green-400: #{$pf-chart-color-green-400};
+  --pf-chart-color-green-500: #{$pf-chart-color-green-500};
+
+  // cyan
+  --pf-chart-color-cyan-100: #{$pf-chart-color-cyan-100};
+  --pf-chart-color-cyan-200: #{$pf-chart-color-cyan-200};
+  --pf-chart-color-cyan-300: #{$pf-chart-color-cyan-300};
+  --pf-chart-color-cyan-400: #{$pf-chart-color-cyan-400};
+  --pf-chart-color-cyan-500: #{$pf-chart-color-cyan-500};
+
+  // purple
+  --pf-chart-color-purple-100: #{$pf-chart-color-purple-100};
+  --pf-chart-color-purple-200: #{$pf-chart-color-purple-200};
+  --pf-chart-color-purple-300: #{$pf-chart-color-purple-300};
+  --pf-chart-color-purple-400: #{$pf-chart-color-purple-400};
+  --pf-chart-color-purple-500: #{$pf-chart-color-purple-500};
+
+  // yellow
```

Comment:
should we be calling this yellow since the colors are labeled gold?

Reply:
yep im fine with that @gdoyle1 ?